content/2_Bronze/Intro_Greedy.mdx

```diff
@@ -179,3 +180,65 @@ programmers refer to this as "Proof by AC," or "Proof by Accepted."
 <Problems problems="general" />
 
 <!-- Anything else major that needs to be added? Bronze Ad Hoc is not very complicated. -->
+
+## Quiz
+	
+<Quiz>
+  <Quiz.Question>
+    What's a greedy algorithm?
+    <Quiz.Answer>
+      An algorithm which takes back choices in order to find the most optimal solution.
+      <Quiz.Explanation>
+        Incorrect. A greedy algorithm doesn't take back choices. Instead, it directly constructs an answer.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      An algorithm which makes the locally optimal choice at each stage.
+      <Quiz.Explanation>
+        Correct. This allows a greedy algorithm to be very efficient and often easier to code.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      An algorithm which tries every possible combination in order to find the most optimal outcome.
+      <Quiz.Explanation>
+        Incorrect. This is known as [complete search](https://usaco.guide/bronze/intro-complete) or [brute force](https://usaco.guide/bronze/intro-complete).
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Why would a greedy algorithm be preferred over a complete search algorithm when faced with a large input size? 
+    <Quiz.Answer correct>
+      Since greedy algorithms directly construct an answer, greedy algorithms are generally more efficient.
+      <Quiz.Explanation>
+        Correct.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Since finding an optimal choice is often easier than doing casework for all pairs, greedy algorithms are easier to implement.
+      <Quiz.Explanation>
+        Incorrect. This varies for every problem.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Greedy algorithms are much easier to debug.
+      <Quiz.Explanation>
+        Incorrect. This varies for every problem.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Why should you learn how to greedily think about problems?
+    <Quiz.Answer correct>
+      It provides a new outlook on a problem, and could help you solve a problem much faster!
+      <Quiz.Explanation>
+        Correct. All of these are viable answers.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      It's a substitute for other algorithms.
```

Comment:
i would say it can be at the very least because this might wrongly imply that it could substitute a large number of algorithms (usually certain preconditions have to be met)

Reply:
as in "it can be a substitute for algorithms"